drivers/i2c/stm32f7_i2c.c

```diff
@@ -494,8 +494,26 @@ static int stm32_i2c_xfer(struct udevice *bus, struct i2c_msg *msg,
 			  int nmsgs)
 {
 	struct stm32_i2c_priv *i2c_priv = dev_get_priv(bus);
+	static bool skip = true;
 	int ret;
 
+	if (msg->addr == 0x48) {
```

Comment:
Please add a comment saying what is supposed to be 0x48.

Reply:
I dont think it is needed...everyone knows 0x48 is the SE05x...and the commits also indicates it is a workaround for the SE05x. IT is impossible not to know what it is...

Reply:
We know now, we might not know later on when reading the code :-)

Just add a comment to facilitate rebasing later on.

Reply:
ummm...ok, but just because I trust your memory more than mine